src/routes/register/Register.jsx

```diff
@@ -45,11 +53,122 @@ const Register = () => {
   const handleSubmit = async (e) => {
     e.preventDefault();
     const response = await networkAdapter.put('/api/users/register', formData);
+
     if (response && response.errors && response.errors.length < 1) {
       setSuccessMessage(REGISTRATION_MESSAGES.SUCCESS);
       setShowSuccess(true);
       setTimeout(() => navigate('/login'), 2000);
     }
+
+    /*
+      Firstname input validation
+    */
```

Comment:
Prefer single line comment here and in other cases as well.
```suggestion
    // Firstname input validation
```